mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -48,7 +49,17 @@ public CassandraAttachmentModule() {
                     .addColumn(CassandraAttachmentTable.TYPE, text())
                     .addColumn(CassandraAttachmentTable.SIZE, bigint())
                     .withOptions()
-                    .comment("Holds attachment for fast attachment retrieval")));
+                    .comment("Holds attachment for fast attachment retrieval")),
+            new CassandraTable(CassandraAttachmentV2Table.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraAttachmentV2Table.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraAttachmentV2Table.ID, text())
```

Comment:
text ?

Reply:
Text because it was the previous structure.

When we hash content, we have it as text. Nobody took time to got something like (let's say) UUID. I will have a look.

Reply:
I can not change the format of Attachment ID easily:
 - They are mentionned in **message** table.
 - For migration purposes we need attachment Ids in both tables to be the same.

One solution would be:
 - Store the hash string (attachmentId) in a column.
 - Use its UUID as primary key.

Would that be OK for you?

Reply:
:+1: comment took into account in the scope of this PR